gcn/notices/swift/bat/guano/alert.schema.json

```diff
@@ -1,26 +1,40 @@
 {
   "$id": "https://gcn.nasa.gov/schema/gcn/notices/swift/bat/guano/alert.schema.json",
   "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Swift/BAT-GUANO Alert",
+  "description": "Candidate gamma-ray transient reported from the BAT-GUANO targeted search",
   "type": "object",
-  "properties": {
-    "alert_reporter": {
-      "$ref": "/schema/gcn/notices/core/Reporter.schema.json"
-    },
-    "alert_info": {
+  "anyOf": [
+    {
       "$ref": "/schema/gcn/notices/core/Alert.schema.json"
     },
-    "event": {
+    {
       "$ref": "/schema/gcn/notices/core/Event.schema.json"
     },
-    "datetime": {
+    {
       "$ref": "/schema/gcn/notices/core/DateTime.schema.json"
     },
-    "localization": {
+    {
       "$ref": "/schema/gcn/notices/core/Localization.schema.json"
     },
-    "statistics": {
+    {
       "$ref": "/schema/gcn/notices/core/Statistics.schema.json"
+    },
+    {
+      "$ref": "/schema/gcn/notices/core/Reporter.schema.json"
+    },
+    {
+      "$ref": "/schema/gcn/notices/core/AdditionalInfo.schema.json"
+    }
+  ],
+  "properties": {
+    "follow_up_event": {
+      "type": "string",
+      "description": "Name or trigger time of the external trigger that launched the search."
+    },
+    "follow_up_type": {
```

Comment:
Similarly, we have a field for this one in the Reporter.schema as messenger. Could you please replace it with already existing fields in the core?

Reply:
This is inadequate, because I already use messenger:EM for the GRB notice, and need a way of denoting the messeger of the source trigger.

Reply:
Ok, but then name of event will cover GRB.

Reply:
How do you figure?

Also, consider, GRB is just one example usage 
 For example S190814 is pretty opaque to event type.